Content/Library/docsTool/templates/partNested.fs

```diff
@@ -0,0 +1,94 @@
+module PartNested
+open System
+open Fable.React
+open Fable.React.Props
+open FSharp.MetadataFormat
+
+
+let partNested (types : Type array) (modules : Module array) =
+    [
+        if types.Length > 0 then
+            yield table [ Class "table" ] [
+                thead [] [
+                    tr [] [
+                        th [Class "fit"] [
+
+                        ]
+                        th [] [
+                            str "Type"
+                        ]
+                        th [] [
+                            str "Description"
+                        ]
+                    ]
+                ]
+                tbody [] [
+                    for t in types do
+                        yield tr [] [
+                            td [] [
+                                Helpers.createAnchorIcon t.Name
+                            ]
+                            td [Class "type-name"] [
+                                a [Href (sprintf "%s.html" t.UrlName)] [
+                                    str t.Name
+                                ]
+                            ]
+                            td [Class "xmldoc"] [
+                                if t.IsObsolete then
```

Comment:
I've seen this obsolete chunk in a few places, is it worth it to extract it for uniform representation?